rest/data/task.go

```diff
@@ -84,12 +86,13 @@ func FindTasksByProjectAndCommit(opts task.GetTasksByProjectAndCommitOptions) ([
 				break
 			}
 		}
-		if !found {
-			return []task.Task{}, gimlet.ErrorResponse{
-				StatusCode: http.StatusNotFound,
-				Message:    fmt.Sprintf("task '%s' not found", opts.StartingTaskId),
-			}
-		}
+		grip.DebugWhen(!found, message.Fields{
```

Comment:
Can we include any other opts that we may be filtering on?

Reply:
printing opts